src/config/containers-and-vms/chroot.md

```diff
@@ -0,0 +1,80 @@
+# Creating and using chroots
+
+Similar to container solutions like docker or podman, chroots can be set up and
+used for many purposes, including:
+
+- running glibc software on musl (and vice versa)
+- running software in a more controlled or sandboxed environment
+- creating a rootfs for bootstrapping a system
+
+## Creation
+
+Create a directory that will contain the chroot, and install a base system in it
+via the `base-voidstrap` package:
+
+```
+$ mkdir <chroot_dir>
+$ XBPS_TARGET_ARCH=<chroot_arch> xbps-install -M -r <chroot_dir> -R <repository> base-voidstrap
+```
+
+The `<repository>` varies depending on architecture:
+
+- `aarch64`, `aarch64-musl`:
+   `https://repo-default.voidlinux.org/current/aarch64`
+- any other `musl` architecture:
+   `https://repo-default.voidlinux.org/current/musl`
+- any other architecture: `https://repo-default.voidlinux.org/current`
+
+## Usage
+
+### xchroot
+
+[`xchroot(1)`](https://man.voidlinux.org/xchroot.1) (from `xtools`) can be used
+to automatically set up and enter the chroot.
+
+### Manual Method
+
+Alternatively, this process can be done manually.
+
+If network access is required, copy `/etc/resolv.conf` into the chroot;
+`/etc/hosts` may need to be copied as well.
+
+Several directories then need to be mounted as follows:
+
+```
+# mount -t proc none <chroot_dir>/proc
+# mount -t sysfs none <chroot_dir>/sys
+# mount --rbind /dev <chroot_dir>/dev
+# mount --rbind /run <chroot_dir>/run
+```
+
+Use [chroot(1)](https://man.voidlinux.org/chroot.1) to change to the new root,
+then run programs and do tasks as usual. Once you've finished using it, unmount
+the chroot using [umount(8)](https://man.voidlinux.org/umount.8).
+
+### Rootless Alternatives
+
+There are several other methods for using chroots that do not require root
+access.
+
+#### xbps
+
+[xbps-uchroot(1)](https://man.voidlinux.org/xbps-uchroot.1) and
+[xbps-uunshare(1)](https://man.voidlinux.org/xbps-uunshare.1) are part of
+`xbps`. `xbps-uchroot` can also used to create an overlayfs layer for making
+temporary changes within the chroot. `xbps-uunshare` uses
+[user_namespaces(7)](https://man.voidlinux.org/user_namespaces.7) for isolation.
```

Comment:
I don't think `isolation` is the right word here.

Regarding these tools, pinging @Duncaen and @Chocimier to ask what they think of having them as recommendations for tasks other than building packages.